dsp/dsp.h

```diff
@@ -10,296 +8,6 @@
 #include <vector>
 
 #include "IPlugConstants.h"
-#include <Eigen/Dense>
```

Comment:
My understanding is that this is a straight move over to `NAM/dsp.h`. Is that correct? Let me know if there are any other changes.

Reply:
It moves over the NAM-specific bits of dsp.h to NAM/dsp.h. So the NAM-specific stuff then gets removed from dsp/dsp.h - it just has the base class for the noise gate, IR and EQ  stuff.

Reply:
There is only one other change. The NAM/dsp.h code removes the IPlug dependency - which just amounts to substituting "double" for "iplug::sample"